test/Microsoft.ML.Core.Tests/UnitTests/TestEntryPoints.cs

```diff
@@ -629,46 +638,681 @@ public void EntryPointExecGraphCommand()
-        //    var fastForest = new FastForestClassification(Env, new FastForestClassification.Arguments());
-        //    var rmd = RoleMappedData.Create(splitOutput.TrainData[0],
-        //        RoleMappedSchema.CreatePair(RoleMappedSchema.ColumnRole.Feature, "Features"),
-        //        RoleMappedSchema.CreatePair(RoleMappedSchema.ColumnRole.Label, "Label"));
-        //    fastForest.Train(rmd);
-        //    var ffModel = new PredictorModel(Env, rmd, splitOutput.TrainData[0], fastForest.CreatePredictor());
-        //    var calibratedFfModel = Calibrate.Platt(Env,
-        //        new Calibrate.NoArgumentsInput() { Data = splitOutput.TestData[0], UncalibratedPredictorModel = ffModel }).PredictorModel;
-        //    var twiceCalibratedFfModel = Calibrate.Platt(Env,
-        //        new Calibrate.NoArgumentsInput() { Data = splitOutput.TestData[0], UncalibratedPredictorModel = calibratedFfModel }).PredictorModel;
-        //    var scoredFf = ScoreModel.Score(Env, new ScoreModel.Input() { Data = splitOutput.TestData[2], PredictorModel = twiceCalibratedFfModel }).ScoredData;
-        //}
+        [Fact]
+        public void EntryPointCalibrate()
+        {
+            var dataView = GetBreastCancerDataView();
+
+            var splitOutput = CVSplit.Split(Env, new CVSplit.Input { Data = dataView, NumFolds = 3 });
+
+            var lrModel = LogisticRegression.TrainBinary(Env, new LogisticRegression.Arguments { TrainingData = splitOutput.TestData[0] }).PredictorModel;
+            var calibratedLrModel = Calibrate.FixedPlatt(Env,
+                new Calibrate.FixedPlattInput { Data = splitOutput.TestData[1], UncalibratedPredictorModel = lrModel }).PredictorModel;
+
+            var scored1 = ScoreModel.Score(Env, new ScoreModel.Input() { Data = splitOutput.TestData[2], PredictorModel = lrModel }).ScoredData;
+            scored1 = ScoreModel.SelectColumns(Env, new ScoreModel.ScoreColumnSelectorInput() { Data = scored1, ExtraColumns = new[] { "Label" } }).OutputData;
+
+            var scored2 = ScoreModel.Score(Env, new ScoreModel.Input() { Data = splitOutput.TestData[2], PredictorModel = calibratedLrModel }).ScoredData;
+            scored2 = ScoreModel.SelectColumns(Env, new ScoreModel.ScoreColumnSelectorInput() { Data = scored2, ExtraColumns = new[] { "Label" } }).OutputData;
+
+            Assert.Equal(4, scored1.Schema.ColumnCount);
+            CheckSameValues(scored1, scored2);
+
+            var input = new Calibrate.NoArgumentsInput() { Data = splitOutput.TestData[1], UncalibratedPredictorModel = lrModel };
+            calibratedLrModel = Calibrate.Platt(Env, input).PredictorModel;
+            calibratedLrModel = Calibrate.Naive(Env, input).PredictorModel;
+            calibratedLrModel = Calibrate.Pav(Env, input).PredictorModel;
+
+            // This tests that the SchemaBindableCalibratedPredictor doesn't get confused if its sub-predictor is already calibrated.
```

Comment:
>tests [](start = 20, length = 5)

I don't see any tes0s statements below. Does it mean if there is any problem an exception will be thrown?